### PR TITLE
Improve moment-select so it doesn't ovveride current date

### PR DESCRIPTION
**Pull request by wytrych**
Small bugfix

### CODE REVIEW

**Comment by wytrych** [comment]
@incuna/js Please review

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
-# 8.1.0
+#upcoming
+
+* BUGFIX: moment-select would ovverride any other date information expect for hours and minutes
```
Comment (Minglee01) [inline CHANGELOG.md]: override ?

File: dist/angular-input-fields.js [hunk 2]
```diff
@@ -242,23 +242,34 @@
...
-                                timeObject.minutes = minutes;
+                            var newDatetime;
+
+                            if (scope.useDuration) {
+                                newDatetime = moment.duration({
+                                    hours: hours,
+                                    minutes: minutes
+                                });
+                            } else {
+                                newDatetime = cast(scope.model);
```
Comment (Minglee01) [inline dist/angular-input-fields.js]: Whats `cast()`?

Comment (wytrych) [inline dist/angular-input-fields.js]: Line `240`  `var cast = (scope.useDuration) ? moment.duration : moment;`

File: tests/inputFieldsSpec.js [hunk 3]
```diff
@@ -552,8 +572,8 @@
 
                     $rootScope.$digest();
 
-                    expect($(compiledTemplate, 'select[ng-model="hours"] option:nth-child(1)').attr('label')).toBe('00');
-                    expect($(compiledTemplate, 'select[ng-model="hours"] option:nth-child(11)').attr('label')).toBe('10');
+                    expect($(compiledTemplate, 'select[ng-model="hours"] option:nth-child(2)').attr('label')).toBe('00');
+                    expect($(compiledTemplate, 'select[ng-model="hours"] option:nth-child(11)').attr('label')).toBe('09');
```
Comment (wytrych) [inline tests/inputFieldsSpec.js]: I'm not sure why I had to change these, but testing in the browser looks fine.

**Comment by Minglee01** [comment]
ok fix the changlog and its good to go.

File: tests/inputFieldsSpec.js (hunk 3 above)
Comment (wytrych) [inline tests/inputFieldsSpec.js]: I'm just gonna double check this.

Comment (Minglee01) [inline tests/inputFieldsSpec.js]: is it cause we updated karma?

**Comment by wytrych** [comment]
@Minglee01 Fixed, This is really weird today the test didn't need that change. Ffs.

**Pull request merged**